docs/creating-releases.md

```diff
@@ -0,0 +1,20 @@
+# Creating Releases
+
+Developers of cores _outside_ of the [MiSTer-devel Github organization](https://github.com/MiSTer-devel) should create an external manifest/"database" as described in [Custom Databases](custom-databases.md).
+
+## Hosting the Manifest
+
+There are several recommended methods to host your manifest file:
+
+* Commit the manifest directly to the repository - This has the disadvantage of requiring constant commits simultaneously with your core releases
+* Create an orphaned branch (with no normal commits in it) to contain your manifest - This still requires constant commits, but those commits are hidden away in another branch
+
+Releases can be stored on any webserver, including in Github Releases.
+
+If you are using Github, prefer to host in a way that hits Github's CDN, as downloads will be much faster.
+
+> **Warning**: Github API not supported
```

Comment:
This will create a special Github-flavored message box like so: https://github.com/agg23/openfpga-SNES#snes-for-analogue-pocket